### ISSUE TITLE
lldpctl should show POE status 

### ISSUE THREAD

**Issue by stannous** [vincentbernat/lldpd#358]
To get accurate class information,  we'd like **lldpcli** to decode the "Power Status" field.

Below, the tcpdump Power status 0x13f5 should be interpreted and decoded.

> Power Status: 0x13f5
>             00.. .... .... .... = PSE Powering Status: 0
>             ..01 .... .... .... = PD Powered Status: 1
>             .... 00.. .... .... = PSE Power Pairs ext: 0
>             .... ..11 1... .... = DS Pwr Class Ext A: 7
>             .... .... .111 .... = DS Pwr Class Ext B: 7
>             .... .... .... 0101 = Pwr Class Ext: 5

What we see from lldpctl  is this:
```
Interface:    swp1, via: LLDP, RID: 19, Time: 0 day, 00:50:42
  Chassis:     
    ChassisID:    mac 00:04:a3:00:00:11
    SysDescr:     Sifos PSx-3202
  Port:        
    PortID:       mac 00:04:a3:00:00:11
    TTL:          120
    MDI Power:    supported: no, enabled: no, pair control: no
      Device type:  PD
      Power pairs:  signal
      Class:        class 4
      Power type:   2
      Power Source: Primary power source
      Power Priority: low
      PD requested power Value: 40000
      PSE allocated power Value: 40000

```
```
No.     Time           Source                Destination           Protocol Length Info
      2 5.244489       Microchi_00:00:11     LLDP_Multicast        LLDP     85     TTL = 120 SysDesc = Sifos PSx-3202 

Frame 2: 85 bytes on wire (680 bits), 85 bytes captured (680 bits)
    Encapsulation type: Ethernet (1)
    Arrival Time: Oct 16, 2019 17:38:06.467447000 EDT
    [Time shift for this packet: 0.000000000 seconds]
    Epoch Time: 1571261886.467447000 seconds
    [Time delta from previous captured frame: 5.244489000 seconds]
    [Time delta from previous displayed frame: 5.244489000 seconds]
    [Time since reference or first frame: 5.244489000 seconds]
    Frame Number: 2
    Frame Length: 85 bytes (680 bits)
    Capture Length: 85 bytes (680 bits)
    [Frame is marked: False]
    [Frame is ignored: False]
    [Protocols in frame: eth:ethertype:lldp]
    [Coloring Rule Name: Broadcast]
    [Coloring Rule String: eth[0] & 1]
Ethernet II, Src: Microchi_00:00:11 (00:04:a3:00:00:11), Dst: LLDP_Multicast (01:80:c2:00:00:0e)
    Destination: LLDP_Multicast (01:80:c2:00:00:0e)
        Address: LLDP_Multicast (01:80:c2:00:00:0e)
        .... ..0. .... .... .... .... = LG bit: Globally unique address (factory default)
        .... ...1 .... .... .... .... = IG bit: Group address (multicast/broadcast)
    Source: Microchi_00:00:11 (00:04:a3:00:00:11)
        Address: Microchi_00:00:11 (00:04:a3:00:00:11)
        .... ..0. .... .... .... .... = LG bit: Globally unique address (factory default)
        .... ...0 .... .... .... .... = IG bit: Individual address (unicast)
    Type: 802.1 Link Layer Discovery Protocol (LLDP) (0x88cc)
Link Layer Discovery Protocol
    Chassis Subtype = MAC address, Id: 00:04:a3:00:00:11
        0000 001. .... .... = TLV Type: Chassis Id (1)
        .... ...0 0000 0111 = TLV Length: 7
        Chassis Id Subtype: MAC address (4)
        Chassis Id: Microchi_00:00:11 (00:04:a3:00:00:11)
    Port Subtype = MAC address, Id: 00:04:a3:00:00:11
        0000 010. .... .... = TLV Type: Port Id (2)
        .... ...0 0000 0111 = TLV Length: 7
        Port Id Subtype: MAC address (3)
        Port Id: Microchi_00:00:11 (00:04:a3:00:00:11)
    Time To Live = 120 sec
        0000 011. .... .... = TLV Type: Time to Live (3)
        .... ...0 0000 0010 = TLV Length: 2
        Seconds: 120
    System Description = Sifos PSx-3202
        0000 110. .... .... = TLV Type: System Description (6)
        .... ...0 0000 1110 = TLV Length: 14
        System Description: Sifos PSx-3202
    Ieee 802.3 - Power Via MDI
        1111 111. .... .... = TLV Type: Organization Specific (127)
        .... ...0 0001 1101 = TLV Length: 29
        Organization Unique Code: 00:12:0f (Ieee 802.3)
        IEEE 802.3 Subtype: Power Via MDI (0x02)
        MDI Power Support: 0x00
            .... ...0 = Port Class: PD
            .... ..0. = PSE MDI Power: Not supported
            .... .0.. = PSE MDI Power: Disabled
            .... 0... = PSE Pairs Control Ability: No
        PSE Power Pair: 1
        Power Class: 4 (5)
        01.. .... = Power Type: Type 2 PD Device (1)
        ..01 .... = Power Source: 1 PSE
        .... 0111 = Power Priority: Unknown (7)
        PD Requested Power Value: 40.0. Watt
        PSE Allocated Power Value: 40.0. Watt
        DS PD Requested Power Value Mode A: 0
        DS PD Requested Power Value Mode B: 0
        DS PSE Allocated Power Value Alt A: 0
        DS PSE Allocated Power Value Alt B: 0
        Power Status: 0x13f5
            00.. .... .... .... = PSE Powering Status: 0
            ..01 .... .... .... = PD Powered Status: 1
            .... 00.. .... .... = PSE Power Pairs ext: 0
            .... ..11 1... .... = DS Pwr Class Ext A: 7
            .... .... .111 .... = DS Pwr Class Ext B: 7
            .... .... .... 0101 = Pwr Class Ext: 5
        System Setup: 0x34
            .... 010. = Power Type Ext: 2
        PSE Maximum Available Power Value: 0
        Autoclass: 0x00
            .... .0.. = PSE Autoclass support: 0
            .... ..0. = Autoclass Completed: 0
            .... ...0 = Autoclass Request: 0
            0000 0... = Autoclass Reserved: 0x00
        Power down: 0x000000
            0000 00.. .... .... .... .... = Power down request: 0
            .... ..00 0000 0000 0000 0000 = Power down time: 0
    End of LLDPDU
        0000 000. .... .... = TLV Type: End of LLDPDU (0)
        .... ...0 0000 0000 = TLV Length: 0

0000  01 80 c2 00 00 0e 00 04 a3 00 00 11 88 cc 02 07   ................
0010  04 00 04 a3 00 00 11 04 07 03 00 04 a3 00 00 11   ................
0020  06 02 00 78 0c 0e 53 69 66 6f 73 20 50 53 78 2d   ...x..Sifos PSx-
0030  33 32 30 32 fe 1d 00 12 0f 02 00 01 05 57 01 90   3202.........W..
0040  01 90 00 00 00 00 00 00 00 00 13 f5 34 00 00 00   ............4...
0050  00 00 00 00 00                                    .....

```

**Comment by vincentbernat** [comment]
I think this will be solved by #350.

**Comment by stannous** [comment]
Thanks, Vincent. Patch #350 works well.
